Automattic/jetpack
# Podcast player: Moves podcast-player json schema to podcast helper

#### Changes proposed in this Pull Request:

Moves json schema for `/podcast-player` endpoint to the podcast helper library, so the schema can be reused in a global wpcom endpoint (see D54402-code), needed for podcast specific onboarding for unauthenticated users in WordPress.com.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* While editing a post or page, insert the podcast player block
* Input a valid podcast rss feed, and make sure the block works as expected for playing the 5 most recent episodes

#### Proposed changelog entry for your changes:

n/a
